adal/tests/Test.ADAL.NET.UIAutomation/CoreTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using Test.ADAL.NET.UIAutomation.Infrastructure;
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+
+namespace Test.ADAL.NET.UIAutomation
+{
+	public class CoreMobileTests
+	{
+        public static void AcquireTokenTest(ITestController controller)
+		{
+            var user = controller.GetUser(
+                new UserQueryParameters
+                {
+                    IsMamUser = false,
+                    IsMfaUser = false,
+                    IsFederatedUser = false
+                });
+
+            controller.Tap("secondPage");
+            controller.Tap("acquireToken");
+            controller.EnterText("i0116", user.Upn, true);
+            controller.Tap("idSIButton9", true);
+            controller.EnterText("i0118", ((LabUser)user).GetPassword(), true);
+            controller.Tap("idSIButton9", true);
```

Comment:
Do you control these IDs? If so, can you rename them to something more human readable? if not, can you add comments on what each point at? idSIButton9 is not very readable :)

Reply:
The ones that are readable are controllable by me, the others are not. ill add comments.